src/command_modules/azure-cli-role/azure/cli/command_modules/role/_help.py

```diff
@@ -159,27 +153,30 @@
     parameters:
         - name: --role-definition
           type: string
-          short-summary: 'JSON formatted string or a path to a file with such content'
+          short-summary: Description of a role as JSON, or an @<file> containing a JSON description.
```

Comment:
The new text is correct (anything that accepts JSON will work with @{file}) but in this case, the command will sniff a file path without @, so you should retain that.  "JSON description of a role or a path to a JSON file containing the description."